### PR TITLE
Clarify definition of PC and GAS instructions

### PR DESCRIPTION
**Pull request by Gustav-Simonsson**

### CODE REVIEW

File: Paper.tex [hunk 1]
```diff
@@ -1795,13 +1795,13 @@ \subsection{Instruction Set}
...
 \midrule
-0x58 & {\small PC} & 0 & 1 & Get the program counter. \\
+0x58 & {\small PC} & 0 & 1 & Get the value of the program counter \textit{before} it's incremented by this instruction. \\
 &&&& $\boldsymbol{\mu}'_\mathbf{s}[0] \equiv \boldsymbol{\mu}_{pc}$ \\
 \midrule
 0x59 & {\small MSIZE} & 0 & 1 & Get the size of active memory in bytes. \\
 &&&& $\boldsymbol{\mu}'_\mathbf{s}[0] \equiv 32\boldsymbol{\mu}_{i}$ \\
 \midrule
-0x5a & {\small GAS} & 0 & 1 & Get the amount of available gas. \\
+0x5a & {\small GAS} & 0 & 1 & Get the amount of gas available after execution of this instruction.  \\
```
Comment (gavofyork) [inline Paper.tex]: while literally correct, this has the wrong implication: that gas is reduced during the course of the execution of an operation. gas is reduced, *then* the instruction is executed.

**Comment by Gustav-Simonsson** [comment]
@gavofyork updated

File: Paper.tex [hunk 2]
```diff
@@ -1795,13 +1795,13 @@ \subsection{Instruction Set}
 &&&& $J_{\text{\tiny JUMPI}}(\boldsymbol{\mu}) \equiv \begin{cases} \boldsymbol{\mu}_\mathbf{s}[0] & \text{if} \quad \boldsymbol{\mu}_\mathbf{s}[1] \neq 0 \\ \boldsymbol{\mu}_{pc} + 1 & \text{otherwise} \end{cases} $ \\
 &&&& This has the effect of writing said value to $\boldsymbol{\mu}_{pc}$. See section \ref{ch:model}. \\
 \midrule
-0x58 & {\small PC} & 0 & 1 & Get the program counter. \\
+0x58 & {\small PC} & 0 & 1 & Get the value of the program counter \textit{before} it's incremented by this instruction. \\
```
Comment (gavofyork) [inline Paper.tex]: "...prior to the increment corresponding to this instruction."

File: Paper.tex [hunk 3]
```diff
@@ -1795,13 +1795,13 @@ \subsection{Instruction Set}
...
 &&&& $\boldsymbol{\mu}'_\mathbf{s}[0] \equiv \boldsymbol{\mu}_{pc}$ \\
 \midrule
 0x59 & {\small MSIZE} & 0 & 1 & Get the size of active memory in bytes. \\
 &&&& $\boldsymbol{\mu}'_\mathbf{s}[0] \equiv 32\boldsymbol{\mu}_{i}$ \\
 \midrule
-0x5a & {\small GAS} & 0 & 1 & Get the amount of available gas. \\
+0x5a & {\small GAS} & 0 & 1 & Get the amount of available gas, taking into account the cost of this instruction. \\
```
Comment (gavofyork) [inline Paper.tex]: "...including the corresponding reduction for the cost of this instruction."

**Pull request merged**